openlibrary/templates/subjects/tagging_menu.html

```diff
@@ -0,0 +1,25 @@
+$def with (work_ids)
```

Comment:
As you mentioned let's see if ChatGPT can pull out all the styling into a new css file 😍